doc/reference/configuration/cfg_networking.rst

```diff
@@ -69,3 +69,21 @@
     | Type: integer
     | Default: 16320
     | Dynamic: **yes**
+
+.. _cfg_networking-iproto_threads:
+
+.. confval:: iproto_threads
```

Comment:
```suggestion
..   confval:: iproto_threads
```

This was indented with 2 spaces and the text below with 4. As a result, the text is formatted as a blockquote. See https://tarantool-develop.herokuapp.com/en/doc/pgulutzan-gh-2059/reference/configuration/#confval-iproto_threads.

Reply:
I did make a mistake but it doesn't seem to be with confval, which is done the same way that other items in the file are done. However, the text below was was indented with 5 spaces.

Reply:
Oh, it's 5 spaces, indeed. Should be 4 as well.